Levels/NewStarbase/NewStarbase.prefab

```diff
@@ -27727,96 +29736,346 @@
                 },
                 {
                     "op": "replace",
-                    "path": "/ContainerEntity/Components/Component_[9803039979756737014]/Transform Data/Translate/0",
-                    "value": 44.6600341796875
+                    "path": "/ContainerEntity/Components/Component_[11643599802128616532]/Transform Data/Translate/0",
+                    "value": 44.523067474365234
                 },
                 {
                     "op": "replace",
-                    "path": "/ContainerEntity/Components/Component_[9803039979756737014]/Transform Data/Translate/1",
-                    "value": -14.91391944885254
+                    "path": "/ContainerEntity/Components/Component_[11643599802128616532]/Transform Data/Translate/1",
+                    "value": -13.417659759521484
                 },
                 {
                     "op": "replace",
-                    "path": "/ContainerEntity/Components/Component_[9803039979756737014]/Transform Data/Translate/2",
-                    "value": -5.7346696853637695
+                    "path": "/ContainerEntity/Components/Component_[11643599802128616532]/Transform Data/Translate/2",
+                    "value": -1.5001277923583984
                 }
             ]
         },
-        "Instance_[332692577265516]": {
-            "Source": "KB3D_HighTechStreets/Objects/Platform_Tile/Platform_Tile.prefab",
+        "Instance_[34318143886290]": {
+            "Source": "Prefabs/Ground_Teleport_Volume.prefab",
             "Patches": [
+                {
+                    "op": "add",
+                    "path": "/Entities/Entity_[331437450142607]/Components/Component_[6232834962003349701]/PendingComponents/0",
+                    "value": {
+                        "$type": "EditorColliderComponent",
+                        "Id": 9876359112770311095,
+                        "ColliderConfiguration": {
+                            "CollisionLayer": {
+                                "Index": 0
+                            },
+                            "CollisionGroupId": {
+                                "GroupId": "{00000000-0000-0000-0000-000000000000}"
+                            },
+                            "Visible": false,
+                            "Trigger": true,
+                            "Simulated": true,
+                            "InSceneQueries": true,
+                            "Exclusive": true,
+                            "Position": [
+                                0.0,
+                                0.0,
+                                -1.0
+                            ],
+                            "Rotation": [
+                                0.0,
+                                0.0,
+                                0.0,
+                                1.0
+                            ],
+                            "MaterialSlots": {
+                                "Slots": [
```

Comment:
This instance override is what breaks the teleport volume.

Reply:
OK I fixed this and the teleporter is functional now!